### DISCUSSION TITLE
system backup failed

### DISCUSSION THREAD

**Discussion by linpanru** [longhorn/longhorn#5517] [Q&A]
![image](https://user-images.githubusercontent.com/56077231/223893561-426cda6c-50cf-4c18-8c0c-bc13a3782384.png)

i am a new longhorn user, by longhorn ui ,i create a systembackup, but  it failed finally, please help me

**Comment by c3y1huang** [comment]
Can you provide the support bundle?

**Reply by linpanru** [reply]
> Can you provide the support bundle?
soorry, i can not

**Reply by linpanru** [reply]
> > Can you provide the support bundle?
> > soorry, i can not

[supportbundle_e726b14e-fbf8-43d8-809d-d02c4eb73d5f_2023-03-09T02-19-03Z.zip](https://github.com/longhorn/longhorn/files/10927049/supportbundle_e726b14e-fbf8-43d8-809d-d02c4eb73d5f_2023-03-09T02-19-03Z.zip)

**Reply by c3y1huang** [reply]
The system backup creates the temporary zip file within the manager. If the manager restarts, the temporary file goes with it. The support bundle shows several restarts of the manager pods and other pods.

```
NAME                                                              READY   STATUS      RESTARTS         AGE
csi-attacher-dcb85d774-6qp6r                                      1/1     Running     26 (108m ago)    2d16h
csi-attacher-dcb85d774-kgrqg                                      1/1     Running     3 (2d9h ago)     2d16h
csi-provisioner-5d8dd96b57-kh6m6                                  1/1     Running     1 (47h ago)      2d16h
csi-provisioner-5d8dd96b57-mnvhm                                  1/1     Running     26 (108m ago)    2d16h
csi-resizer-6bf6f6f584-fbz4b                                      1/1     Running     4 (2d9h ago)     2d16h
csi-resizer-6bf6f6f584-m7hkv                                      1/1     Running     23 (108m ago)    2d16h
csi-snapshotter-7cb6bf8447-4p9tf                                  1/1     Running     6 (108m ago)     2d16h
csi-snapshotter-7cb6bf8447-bds57                                  1/1     Running     20 (6h37m ago)   2d16h
longhorn-manager-jcwtz                                            1/1     Running     1 (17h ago)      2d16h
longhorn-manager-mrk5s                                            1/1     Running     8 (93m ago)      2d16h
```

To investigate the cause of the restarts, could you check on the previous logs for those pods? ex: `kubectl -n longhorn-system logs longhorn-manager-mrk5s --previous`

**Reply by linpanru** [reply]
> The system backup creates the temporary zip file within the manager. If the manager restarts, the temporary file goes with it. The support bundle shows several restarts of the manager pods and other pods.
> 
> ```
> NAME                                                              READY   STATUS      RESTARTS         AGE
> csi-attacher-dcb85d774-6qp6r                                      1/1     Running     26 (108m ago)    2d16h
> csi-attacher-dcb85d774-kgrqg                                      1/1     Running     3 (2d9h ago)     2d16h
> csi-provisioner-5d8dd96b57-kh6m6                                  1/1     Running     1 (47h ago)      2d16h
> csi-provisioner-5d8dd96b57-mnvhm                                  1/1     Running     26 (108m ago)    2d16h
> csi-resizer-6bf6f6f584-fbz4b                                      1/1     Running     4 (2d9h ago)     2d16h
> csi-resizer-6bf6f6f584-m7hkv                                      1/1     Running     23 (108m ago)    2d16h
> csi-snapshotter-7cb6bf8447-4p9tf                                  1/1     Running     6 (108m ago)     2d16h
> csi-snapshotter-7cb6bf8447-bds57                                  1/1     Running     20 (6h37m ago)   2d16h
> longhorn-manager-jcwtz                                            1/1     Running     1 (17h ago)      2d16h
> longhorn-manager-mrk5s                                            1/1     Running     8 (93m ago)      2d16h
> ```
> 
> To investigate the cause of the restarts, could you check on the previous logs for those pods? ex: `kubectl -n longhorn-system logs longhorn-manager-mrk5s --previous`

![image](https://user-imag
[longhorn-manager-mrk5s-previous.log](https://github.com/longhorn/longhorn/files/10927322/longhorn-manager-mrk5s-previous.log)
es.githubusercontent.com/56077231/223907048-d268c77a-a4bb-4d44-a530-a145f2509fba.png)
there is a panic when i check previous logs by exec kubectl -n longhorn-system logs longhorn-manager-mrk5s --previous

**Reply by linpanru** [reply]
> The system backup creates the temporary zip file within the manager. If the manager restarts, the temporary file goes with it. The support bundle shows several restarts of the manager pods and other pods.
> 
> ```
> NAME                                                              READY   STATUS      RESTARTS         AGE
> csi-attacher-dcb85d774-6qp6r                                      1/1     Running     26 (108m ago)    2d16h
> csi-attacher-dcb85d774-kgrqg                                      1/1     Running     3 (2d9h ago)     2d16h
> csi-provisioner-5d8dd96b57-kh6m6                                  1/1     Running     1 (47h ago)      2d16h
> csi-provisioner-5d8dd96b57-mnvhm                                  1/1     Running     26 (108m ago)    2d16h
> csi-resizer-6bf6f6f584-fbz4b                                      1/1     Running     4 (2d9h ago)     2d16h
> csi-resizer-6bf6f6f584-m7hkv                                      1/1     Running     23 (108m ago)    2d16h
> csi-snapshotter-7cb6bf8447-4p9tf                                  1/1     Running     6 (108m ago)     2d16h
> csi-snapshotter-7cb6bf8447-bds57                                  1/1     Running     20 (6h37m ago)   2d16h
> longhorn-manager-jcwtz                                            1/1     Running     1 (17h ago)      2d16h
> longhorn-manager-mrk5s                                            1/1     Running     8 (93m ago)      2d16h
> ```
> 
> To investigate the cause of the restarts, could you check on the previous logs for those pods? ex: `kubectl -n longhorn-system logs longhorn-manager-mrk5s --previous`

![image](https://user-images.githubusercontent.com/56077231/223907896-9ef2c5cf-7b5b-4fdb-93ca-dede80aa5591.png)
[longhorn-manager-mrk5s-previous.log](https://github.com/longhorn/longhorn/files/10927352/longhorn-manager-mrk5s-previous.log)

there is a panic by check previous log

**Reply by c3y1huang** [reply]
Thanks! this is a bug that needs to be fixed.

**Reply by c3y1huang** [reply]
Actually, this is a known issue in v1.4.0 and included in v1.4.1.
Ref: https://github.com/longhorn/longhorn/issues/5286

**Reply by c3y1huang** [reply]
@linpanru , can you wait for v1.4.1 and try again?

**Reply by linpanru** [reply]
> @linpanru , can you wait for v1.4.1 and try again?

of course，thanks again your help
